gauche/kernels/fingerprint_kernels/base_fingerprint_kernel.py

```diff
@@ -25,6 +25,31 @@ def batch_tanimoto_sim(
     )
 
 
+def minmax_sim(
+    x1: torch.Tensor, x2: torch.Tensor, eps: float = 1e-6
+) -> torch.Tensor:
+    """
+    Non-batch version of minmax similarity.
```

Comment:
@AustinT Is there a batch version of minmax similarity available?

Reply:
Also is there a reference for the term minmax? I looked in Ioffe 2010 and it seems the metric is Jaccard similarity there

Reply:
Seems to match Jaccard similarity from here: 

https://www.iiisci.org/journal/pdv/sci/pdfs/GS315JG.pdf

Reply:
> Also is there a reference for the term minmax? I looked in Ioffe 2010 and it seems the metric is Jaccard similarity there

I think there are many names for this kernel. I saw minmax in "Graph kernels for chemical informatics" by Ralaivola et al (2005)

Reply:
> Is there a batch version of minmax similarity available?

Good question: I checked and it looks like `torch.cdist` can use batch evaluation. I pushed a commit to add this.